test/nlp/unit/test_metadata_extractors.py

```diff
@@ -33,11 +33,30 @@
 from elemeta.nlp.extractors.high_level.out_of_vocabulary_count import OutOfVocabularyCount
 from elemeta.nlp.extractors.high_level.word_count import WordCount
 from elemeta.nlp.extractors.high_level.word_regex_matches_count import WordRegexMatchesCount
+from elemeta.nlp.extractors.high_level.pii_identifier import PII_Identifier
 from elemeta.nlp.extractors import length_check_basic, avg_check_basic
 
 
 # TODO for all check tokenizer difference. example can be between twitter and not. the parse isn't differently
 
+@pytest.mark.parametrize(
+    "name, text, required_PII",
+    [
```

Comment:
Please add more entities converge to see that we do detect all the entities and we support detection of multiple entities in one text